scarb/tests/build_starknet_contract_allowed_libfuncs.rs

```diff
@@ -103,13 +105,13 @@ fn check_false() {
         .build(&t);
 
     Scarb::quick_snapbox()
-        .arg("build") // TODO(#137): Change build to check for faster and lighter test.
+        .arg("check")
         .current_dir(&t)
         .assert()
         .success()
         .stdout_matches(indoc! {r#"
-        [..] Compiling hello v0.1.0 ([..])
-        [..]  Finished release target(s) in [..]
+        [..]  Checking hello v0.1.0 ([..])
+        [..]  Finished checking release target(s) in [..]
         "#});
```

Comment:
If `check` does not check this fact, that changing `build` to `check` here makes the test misbehaving, as we want to ensure that the warning does not pop up here.